src/UiTabs.vue

```diff
@@ -20,7 +21,7 @@
                     v-for="tab in tabs"
                 >
                     <render :nodes="tab.$slots.header()" v-if="tab.$slots.header"></render>
-                    <template v-if="!tab.$slots.header && hasIcon && Boolean(tab.$slots.icon)" #icon>
+                    <template v-if="!tab.$slots.header && tab.$slots.icon && hasIcon" #icon>
```

Comment:
```suggestion
                    <template v-else-if="tab.$slots.icon && hasIcon" #icon>
```

Reply:
This won't work cause a template with a slot cannot have v-else-if on it. This is the error I get:
`v-else/v-else-if has no adjacent v-if or v-else-if.`
It seems to be a bug in the Vue compiler. A few references:
1. https://github.com/vuejs/eslint-plugin-vue/issues/1316
2. https://github.com/vuejs/core/issues/5210

Reply:
Makes sense.